python/caffe/io.py

```diff
@@ -3,7 +3,11 @@
 from scipy.ndimage import zoom
 from skimage.transform import resize
 
-from caffe.proto import caffe_pb2
+try:
+	# Python3 will most likely not be able to load protobuf
+	from caffe.proto import caffe_pb2
+except:
+	pass
```

Comment:
Can we only do this if actually in Python 3 (check `sys.version_info`, I believe), and warn if protobuf doesn't load? Currently this silences lots of actual problem that can happen with the Python protobuf code.